### PR TITLE
POD-717 | Expose setting for setting up git commit signing with SSH in UI

### PR DESCRIPTION
**Pull request by janekbaraniewski**

### CODE REVIEW

File: desktop/src-tauri/Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ tauri-build = { version = "1.2", features = [] }
 
 [dependencies]
 # Tauri
-tauri = { version = "1.2.4", features = [ "dialog-save",
+tauri = { version = "1.2.4", features = [ "updater", "dialog-save",
```
Comment (pascalbreuninger) [inline desktop/src-tauri/Cargo.toml:15]: can you remote this again pls?

File: desktop/src/client/client.ts [hunk 2]
```diff
@@ -63,7 +63,7 @@ type TChannelName = keyof TChannels
 type TClientEventListener<TChannel extends TChannelName> = (payload: TChannels[TChannel]) => void
 type TClientSettings = Pick<
   TSettings,
-  "debugFlag" | "additionalCliFlags" | "dotfilesUrl" | "additionalEnvVars"
+  "debugFlag" | "additionalCliFlags" | "dotfilesUrl" | "additionalEnvVars" | "sshKeyPath"
```
Comment (pascalbreuninger) [inline desktop/src/client/client.ts:66]: You'll need to wire it up in the client itself, otherwise it won't have an effect

Comment (janekbaraniewski) [reply desktop/src/client/client.ts:66]: yup yup 👍

File: desktop/src-tauri/Cargo.toml (hunk 1 above)
Comment (janekbaraniewski) [reply desktop/src-tauri/Cargo.toml]: 👍

**Review by pascalbreuninger — APPROVED** [review]

**Pull request merged**